lib/job.js

```diff
@@ -11,7 +11,11 @@ function Job(queue, jobId, data, options) {
   this.id = jobId;
   this.progress = 0;
   this.data = data || {};
-  this.options = options || {};
+  this.options = options || {
+    attemptsMade: 0,
```

Comment:
can we infer this from the length of `stacktraces` or something to avoid extra memory usage?